django/db/models/fields/composite.py

```diff
@@ -13,6 +15,11 @@
 from django.utils.functional import cached_property
 
 
+class AttributeSetter:
+    def __init__(self, name, value):
+        setattr(self, name, value)
```

Comment:
This is a copy of `django.contrib.postgres.fields.utils.AttributeSetter` 
I can pull this out and put this in `django.db.models.models.utils` if we want